DeAceroDLabs/marketplace-frontend
# [DMP-611] Mark product property as selected

## What is done

[[DMP-611] Marcar propiedad como seleccionada](https://deacero.atlassian.net/browse/DMP-611)

## How is done

1. Added a new state in the ProductDetail component to store the selected properties.
2. Added a conditional to check wether a property has been selected or not.

- Modified the PaymentButton and commented the action of it because this was decreasing the code coverage and creating tests for this is not that important right now as its use is still ambiguous.

## How to test

1. Run the project via `npm start`
4. Select a product from the home screen.
5. Wait for the properties to load.
6. Select any property. The status of the button should change to "active".

(You can select all the properties you want for now, it won't validate the correct combination)

## Evidence

Add images or screen  recordings on how the feature was working after and before the fix (if applies)
